.github/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### New features since last release
 
+* `lightning.qubit` supports mid-circuit measurements.
+  [(#621)](https://github.com/PennyLaneAI/pennylane-lightning/pull/621)
+
```

Comment:
The PR context is excellent, but I would suggest updating the changes and the changelog as follows
```suggestion
* Add `collapse` and `normalize` methods to the `StateVectorLQubit` classes, enabling "branching" of the wavefunction. Add methods to create and seed an RNG in the `Measurements` modules.
  [(#645)](https://github.com/PennyLaneAI/pennylane-lightning/pull/645)

```